processing/missing_value_format.py

```diff
@@ -71,6 +74,139 @@ def check_missing_values_col(self, fname, header,
+            drop_line_count = 0
+            header_set = (
+                'timestamp_start', 'timestamp', 'time', 'date', 'year',
+                'start_timestamp', 'doy', 'dtime', 'hrmin',
+                'timestamp_end', 'end_timestamp', 'time_start', 'start_time')
+            while not found_headers:
+                try:
+                    # if readline encounters an invalid UTF8 char it
+                    # tosses the remainder of the 8k block of data
+                    line = f.readline()
+                except UnicodeDecodeError:
+                    if first_bad_char:
+                        _log.warning(
+                            'found invalid UTF8 character when looking for '
+                            f'headers on line {drop_line_count}, '
+                            'trying cp1252')
+                        drop_line_count = 0
+                        f.close()
+                        f = open(fname,
+                                'r', encoding='cp1252')
+                        first_bad_char = False
+                        continue
+                    else:
+                        msg = ('File contains invalid UTF8 and cp1252 '
+                            'characters. Unknown file encoding. '
+                            'Autocorrection FAILED.')
+                        self.append_status_msg_parts('fatal', msg)
+                        return None, None
+                if line == '':
+                    # End of File
+                    break
+                tokens = line.rstrip().split(',')
+                # note that strip and strip(string.whitespace) aren't
+                # exactly the same but FPin files are supposed to be
+                # ascii files so they should yield the same result
+                tokens = [h.strip(string.whitespace + '"') for h in tokens]
+                head = [t for t in tokens if t.lower() in header_set]
+                if len(head) > 0:
+                    # found header line
+                    found_headers = True
+                    break
+                drop_line_count += 1
+
+            headers = tokens
+            data = f.readlines()
+        return headers, data
+
+    def has_invalid_values(self, data):
+        s = ''.join(data).replace('\n', ',').lower()
+        invalid_char = ('i', '!', 'e', '+')
+        if set(s).intersection(invalid_char):
+            return True
+        return False
+
+    def find_invalid_values(self, headers, data, check_log):
+            for ln_num, line in enumerate(data):
+                line = line.lower()
+                tokens = self._tokenize(line.strip('\n'))
+                tokens, has_quotes_removed = self._strip_quotes(tokens)
+                tokens, has_whitespace_removed = self._strip_whitespace(tokens)
+                if has_whitespace_removed:
+                    n_lines_whitespace_removed += 1
+                if has_quotes_removed:
+                    n_lines_quotes_removed += 1
```

Comment:
It's great to track this as I don't believe we are checking for elsewhere except in the file fixer.
So we need to add another logger to track and report the white spaces and quotes.